client/data/geo/use-geolocation-query.ts

```diff
@@ -13,8 +13,8 @@ export const useGeoLocationQuery = ( options = {} ) =>
 	useQuery< GeoLocationData >( {
 		queryKey: [ 'geo' ],
 		queryFn: () =>
-			globalThis
-				.fetch( 'https://public-api.wordpress.com/geo/' )
-				.then( ( response ) => response.json() ),
+			fetch( 'https://public-api.wordpress.com/geo/' ).then( ( response ) => response.json() ),
+		refetchInterval: Infinity,
+		meta: { persist: false },
```

Comment:
Just to confirm, the intent is to:

- Fetch once initially
- Never fetch again in that window